src/Symfony/Component/Console/Question/Question.php

```diff
@@ -188,7 +188,6 @@ public function setAutocompleterCallback(callable $callback = null): self
     /**
      * Sets a validator for the question.
      *
-     * @param callable|null $validator
      *
```

Comment:
can extra blank lines be removed by the tool too?

Reply:
hmm shouldn't fabbot.io catch those ?

I ran php-cs-fixer with only `'no_superfluous_phpdoc_tags' => ['allow_mixed' => true],` as a rule to speed up things, so others rules from `.php_cs.dist` might handle these.

What's clear is that the `no_superfluous_phpdoc_tags` doesn't handle it itself after it's cleaning

I'll check my guess and see if there's even another rule we can add that can do this check if not in `.php_cs.dist` file already